frontend/client/components/Proposal/index.less

```diff
@@ -101,7 +100,11 @@
           height: 3rem;
 
           & .ant-input-group {
-            width: inherit
+            width: inherit;
+          }
+
+          & > * + * {
```

Comment:
Whoa, this selector's a little wonky. Why do we need this?

Reply:
Now styling those components directly with a separate class name.